instrumentation/sidekiq/README.md

```diff
@@ -33,7 +33,16 @@ end
 
 Example usage can be seen in the `./example/sidekiq.rb` file [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/sidekiq/example/sidekiq.rb)
 
-## How can I get invovled?
+## Development
+
+You'll need Redis installed locally to run the test suite. Once you've
+installed it, start it with the included configuration:
+
+```
+redis-server test/redis.conf
+```
```

Comment:
While we can't auto-install a local Redis for devs, it'd be nice if we started (and stopped) one on a non-default port for the test suite, e.g. in the Rakefile. It's tedious for devs to remember to start (and stop) a Redis manually when working on this instrumentation, or when running test suites for all gems in this project.